README.md

```diff
@@ -1,5 +1,9 @@
 # HEPData pyhf extractor
 
+![GH Actions status][badge-actions-status]
```

Comment:
Better to link the badge to:
https://github.com/HEPData/hepdata-pyhf-extractor/actions?query=workflow%3A%22Continuous%20Integration%22+branch%3Amain

Reply:
I added the branch notation following the official [GitHub Actions badges guide](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter), in 4d6e800

Reply:
But if you click the badge on the rendered [README.md](https://github.com/HEPData/hepdata-pyhf-extractor/blob/fix-ci-branch/README.md) you just see an image of the badge.  That's not very useful.  It would be better if clicking the badge displayed the relevant workflows, e.g.
```markdown
[![GH Actions status][badge-actions-status]][badge-actions-workflows]
```
and define:
```markdown
[badge-actions-workflows]: https://github.com/HEPData/hepdata-pyhf-extractor/actions?query=workflow%3A%22Continuous%20Integration%22+branch%3Amain
```
This is what's done in the [README.rst](https://github.com/scikit-hep/pyhf/blob/master/README.rst) file for `pyhf` and is similar to what you're already doing for Coverage.

Reply:
Oh, I see. I misunderstood.

I thought you were just referring to the badge image URL, instead of the link.

Reply:
Done in https://github.com/HEPData/hepdata-pyhf-extractor/pull/4/commits/b8748b3a84a3482d7870b2ac5e289b2f9a19b9d4